robottelo/ui/job_template.py

```diff
@@ -0,0 +1,147 @@
+        if snippet:
+            self.click(tab_locators['job.tab_type'])
+            self.assign_value(locators['job.template_snippet'], snippet)
+
+    def create(self, name, template_content, template_type=None, default=None,
+               audit_comment=None, job_category=None, description_format=None,
+               provider_type=None, effective_user_value=None,
+               current_user=None, overridable=None, snippet=None):
+        """Creates a job template from UI."""
+        self.click(locators['job.template_new'])
+        self.click(tab_locators['tab_primary'])
+        self.assign_value(locators['job.template_name'], name)
+        if template_type == 'file':
+            self.wait_until_element(
+                locators['job.template_file']).send_keys(template_content)
+            self.handle_alert(True)
+        elif template_type == 'input':
+            self.assign_value(locators['job.template_input'], template_content)
+        self._configure_job_template(
+            default=default, audit_comment=audit_comment,
+            job_category=job_category, description_format=description_format,
+            provider_type=provider_type,
+            effective_user_value=effective_user_value,
+            current_user=current_user, overridable=overridable,
+            snippet=snippet
+        )
+        self.click(common_locators['submit'])
+
+    def update(self, name, new_name=None, template_content=None,
+               template_type=None, default=None, audit_comment=None,
+               job_category=None, description_format=None, provider_type=None,
+               effective_user_value=None, current_user=None, overridable=None,
+               snippet=None):
+        """Updates a job template from UI."""
+        self.click(self.search(name))
+        self.click(tab_locators['tab_primary'])
+        if new_name:
+            self.assign_value(locators['job.template_name'], new_name)
+        if template_content:
+            if template_type == 'file':
+                self.assign_value(
+                    locators['job.template_file'], template_content)
+                self.handle_alert(True)
+            elif template_type == 'input':
+                self.assign_value(
+                    locators['job.template_input'], template_content)
+        self._configure_job_template(
+            default=default, audit_comment=audit_comment,
+            job_category=job_category, description_format=description_format,
+            provider_type=provider_type,
+            effective_user_value=effective_user_value,
+            current_user=current_user, overridable=overridable,
+            snippet=snippet
+        )
+        self.click(common_locators['submit'])
+
+    def add_input(self, job_template_name, name, required=None,
+                  input_type='User input', advanced=None, options=None,
+                  description=None):
+        """Add new input to existing job template"""
```

Comment:
Does this function defined to support only `User input` input_type ? Just curious about other input types. In any case if this function going to support only `User input` type then I think we should update the docstrings too. thanks